docs/marionette.object.md

```diff
@@ -1,8 +1,6 @@
 # Marionette.Object
 
-A base class which other classes can extend from.
-Object incorporates many backbone conventions and utilities
-like `initialize` and `Backbone.Events`.
+Object incorporates backbone conventions `initialize` and `Backbone.Events`.
```

Comment:
I removed the "A base class which other classes can extend from." since it is no longer true, but now this class doesn't have a description.. what should it be?  My suggestion would be "Essentially and Application without a region that has no particular use case" ;-p  
Seriously though, perhaps the description can come with the rename, but I don't know what this is for.